src/app/frontend/logs/component.js

```diff
@@ -305,6 +305,15 @@ export class LogsController {
     return 'timer_off';
   }
 
+  /**
+   * Return the link to download the log file
+   * @export
+   * @returns {string}
```

Comment:
s/returns/return

`returns` is not recognized by google closure compiler so it is just skipped.